megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -1232,6 +1232,10 @@ else if (wtype.getAtClass() == WeaponType.CLASS_LBX_AC) {
 		                || game.getBoard().getHex(te.getPosition()).containsTerrain(Terrains.ROUGH))) {
 		        toHit.addModifier(+1, "urban guerrilla");
 		    }
+		    if (te.getCrew().getOptions().booleanOption("shaky_stick")
```

Comment:
I don't think this check is sufficient.  According to this, grounded ASF wouldn't get the shaky_stick penalty.  There's a method somewhere, I think in Compute, that will return whether something is airborne.  This should be an airborne check, and not a unit type check.